src/PIL/IcnsImagePlugin.py

```diff
@@ -70,7 +69,7 @@ def read_32(fobj, start_length, size):
                 byte = fobj.read(1)
                 if not byte:
                     break
-                byte = i8(byte)
+                byte = byte[0]
```

Comment:
Is it just me, or is `byte = byte[0]` not as clear as `byte = i8(byte)` about what is happening here?

Reply:
You are right that `i8()` looks more clear at first glance. Mostly because indexing had changed from Py2 to Py3 for bytes. However, this saves one function call and type checking on each value. Also, IDEs handle types correctly and show variables as int.

Reply:
As an alternative, it is possible to change i8 and use it everywhere for consistency:

```python
def i8(c, o=0):
    return c[o]
```